ClientLib/src/main/java/com/o3dr/services/android/lib/coordinate/LatLongAlt.java

```diff
@@ -14,24 +15,34 @@
      * Stores the altitude in meters.
      */
     private double mAltitude;
+    private Frame mFrame;
+
+    public LatLongAlt() {
+        super();
+        mAltitude = 0.0;
+        mFrame = Frame.GLOBAL_RELATIVE;
+    }
 
-    public LatLongAlt(double latitude, double longitude, double altitude) {
+    public LatLongAlt(double latitude, double longitude, double altitude, Frame frame) {
```

Comment:
I di that for a reason in that it exposes the parts in the code that sets the frame.  I.e you can see explicitly what frame is being assumed. You don't want to accidentally put abs alt into relative. It would seem like a flyaway.